scapy/contrib/isotp.uts

```diff
@@ -1502,6 +1554,8 @@ assert(result_data == isotp.data)
 
 = Compatibility with isotprecv - extended addresses
 exit_if_no_isotp_module()
+exit_if_no_can_sockets()
```

Comment:
I would rather edit UTscapy to exclude a tag (you can set a tag to a file, so that would exclude the file), when the dependency is missing